sdk/monitor/Azure.Monitor.Query/src/LogsQueryClientAudience.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Monitor.Query
+{
+    /// <summary>
+    /// Cloud audiences available for <see cref="LogsQueryClient"/>.
+    /// </summary>
+    public readonly partial struct LogsQueryClientAudience : IEquatable<LogsQueryClientAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="LogsQueryClientAudience"/> with a given value.
+        /// </summary>
+        public LogsQueryClientAudience(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        /// <summary> Azure Public Cloud. </summary>
+        public static LogsQueryClientAudience AzurePublicCloud { get; } = new("https://api.loganalytics.io");
```

Comment:
Are there not any other clouds that support this service?

Reply:
We are still figuring out what audiences for those are :) I'll add them as soon as we know.